fvwm/virtual.c

```diff
@@ -1376,22 +1371,18 @@ Bool is_pan_frame(Window w)
 		bool is_pf_right = (w == m->PanFrameRight.win);
 
 		if (is_pf_top) {
-			fvwm_debug(__func__, "Window is PanFrame top\n");
 			return True;
 		}
 
 		if (is_pf_bottom) {
-			fvwm_debug(__func__, "Window is PanFrame bottom\n");
 			return True;
 		}
 
 		if (is_pf_left) {
-			fvwm_debug(__func__, "Window is PanFrame left\n");
 			return True;
 		}
 
 		if (is_pf_right) {
-			fvwm_debug(__func__, "Window is PanFrame right\n");
```

Comment:
These could do with staying around for now -- I still see weird things with different sized monitors and configurations where panframes can go wandering to different locations.

Reply:
Okay, they are back. I just do a lot of EdgeScrolling and my logs are getting filled with these messages.